packages/client/components/Snackbar.tsx

```diff
@@ -25,7 +25,10 @@ const Modal = styled('div')<{isMeetingRoute: boolean; isDesktop: boolean}>(
     paddingBottom: isMeetingRoute ? 64 : 8,
     position: 'fixed',
     top: 0,
-    width: isMeetingRoute && isDesktop ? `calc(100% + ${NavSidebar.WIDTH}px)` : '100%',
+    width:
+      isMeetingRoute && !isStandupRoute && isDesktop
```

Comment:
-1 the initial idea here was that only meeting routes have sidebar, so we could assume, we need to position the snackbar differently in that case. Right now, standup meeting is an exception, so not all meeting have the same layout. instead of introducing another boolean, I'd suggest replacing `isMeetingRoute` and `isStandupRoute` with just one variable, `hasSidebar`, and initializing it accordingly.